app/services/provider_authorisation.rb

```diff
@@ -20,15 +20,16 @@ def can_manage_users_for_at_least_one_provider?
     )
   end
 
-  def providers_that_actor_can_manage_organisations_for
-    provider_ids = ProviderRelationshipPermissions
-      .where(training_provider: @actor.providers)
-      .or(
-        ProviderRelationshipPermissions.where(
-          ratifying_provider_id: @actor.providers,
-        ),
-      )
-      .pluck(:ratifying_provider_id, :training_provider_id).flatten
+  def providers_that_actor_can_manage_organisations_for(with_set_up_permissions: false)
+    scope = ProviderRelationshipPermissions.where(training_provider: @actor.providers).or(
+      ProviderRelationshipPermissions.where(
+        ratifying_provider_id: @actor.providers,
+      ),
+    )
+
+    scope = scope.where.not(setup_at: nil) if with_set_up_permissions
+
+    provider_ids = scope.pluck(:ratifying_provider_id, :training_provider_id).flatten
```

Comment:
Consider extracting the code required to retrieve the manageable provider ids to private methods so there is more clarity about what's going on in this method.